src/routes/models/[...model]/thumbnail.png/ModelThumbnail.svelte

```diff
@@ -19,7 +19,7 @@
 	</div>
 
 	<div
-		class="flex h-[200px] w-full flex-col items-center justify-center rounded-b-none bg-yellow-500/10 pb-10 pt-10 text-4xl text-gray-500"
+		class="flex h-[200px] w-full flex-col items-center justify-center rounded-b-none bg-{PUBLIC_APP_COLOR}-500/10 pb-10 pt-10 text-4xl text-gray-500"
```

Comment:
Oh yes I removed it for simplicity :) also because it means you need to explicitly set it somewhere or it will not be catched by tailwindcss JIT engine).